spec/requests/api/v1/accounts/credentials_spec.rb

```diff
@@ -28,6 +28,20 @@
         locked: true,
       })
     end
+
+    describe 'allows the read:me scope' do
+      let(:scopes) { 'read:me' }
+
+      it 'returns the response successfully' do
+        subject
+
+        expect(response).to have_http_status(200)
+
+        expect(body_as_json).to include({
+          locked: true,
+        })
+      end
```

Comment:
Would it be worth me expanding this to assert that I can't access say `GET /api/v1/accounts/{id}` with this?

Or should we add logic there that allows accessing `GET /api/v1/accounts/{id}` only when `{id}` equals the access token's owner? Returning a 403 if not

Reply:
Ditto around asserting that `PATCH /api/v1/accounts/verify_credentials` isn't possible?

Reply:
You mean `PATCH /api/v1/accounts/update_credentials`? That test certainly wouldn't hurt but I don't think it's a blocker.